test/unit/rules/simple-unless-test.js

```diff
@@ -372,6 +380,11 @@ generateRuleTests({
         '  I think I am a brown stick',
         '{{/unless}}',
       ].join('\n'),
+      fixedTemplate: [
+        '{{#if (not (not isBrown isSticky))}}',
```

Comment:
should we detect such cases and simplify them?

Reply:
Ok `(not (not` chain will be skipped, but if original not have more than one param it will be replaced with `or` helper.
Will also take a look into handling not not in no-negated-conditions

Reply:
related to no-negated-conditions, what should happen with such code?
```hbs
{{#if (not c1 c2)}}{{/if}}
```
If I'm looking corrently it may be broken and result in 
```hbs
{{#unless c1}}{{/unless}}
```

Reply:
@godric3 good catch, can you file that as a bug? Would this be correct and desirable for that?

```hbs
{{#unless c1 c2}}{{/unless}}
```

If there's no good improvement, we could also just not flag a violation in a situation like this.